parser.temper

```diff
@@ -1,6 +1,8 @@
+let Char = Int;
```

Comment:
That I want to type alias this lends support to recommendation by @ben-509 for a custom char type.